### PR TITLE
[TE] rootcause poc - chart tooltip

### PR DESCRIPTION
**Pull request by apucher**

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default Ember.Component.extend({
       return {
         format: {
           title: (d) => {
-            this._onHover(d);
+            this._onHover(moment(d).valueOf());
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: nit: let's save `moment(d)` into a const here.

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-tooltip/template.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+<p>{{hoverTimestamp}}</p>
+
+<div class="container">
+  {{#each urns as |urn|}}
+    <div class="row">
+      <div class="col-xs-2">{{get labels urn}}<br/>{{get values urn}}</div>
+    </div>
+  {{/each}}
+</div>
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-tooltip/template.hbs]: nit: new line at EOF. 

fyi, we're using eslint to catch linting errors in the IDE. I'm using VScode, but i'm pretty sure intellJ also has support for it

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js (hunk 1 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js]: fixed

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-tooltip/template.hbs (hunk 2 above)
Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-tooltip/template.hbs]: fixed

**Pull request merged**